resources/javascripts/login.js

```diff
@@ -0,0 +1,56 @@
+var validation = false;
+
+callback = function (flag, id, value) {
+    validation = flag;
+    document.getElementById(id).innerHTML = value;
+}
+
+
+document.getElementById("submit-login").addEventListener("click", function () {
+    let ele = document.getElementById('login-form');
+    ele.childNodes.forEach(function (element) {

```

Comment:
instead have a classname on all the input fields and get those values from getElementsByClassName